src/MR.AspNetCore.Jobs/JobsOptionsExtensions.cs

```diff
@@ -0,0 +1,41 @@
+﻿using MR.AspNetCore.Jobs;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+
+namespace Microsoft.Extensions.DependencyInjection
+{
+	public static class JobsOptionsExtensions
+	{
+		/// <summary>
+		/// Registers the cron jobs registries within the assembly
+		/// </summary>
+		/// <param name="assembly">Assembly containing classes extending from <see cref="CronJobRegistry"/>.</param>
+		public static void UseCronJobRegistries(this JobsOptions jobsOptions, Assembly assembly)
+		{
+			// get all classes extending cron job registry
+			var types = assembly.GetTypes()
+				.Where(type => typeof(CronJobRegistry)
+				.IsAssignableFrom(type));
```

Comment:
A single indentation here, or just remove this new line,

Reply:
Like this `var types = assembly.GetTypes().Where(type => typeof(CronJobRegistry).IsAssignableFrom(type));` ?

Reply:
Yeah, just a single line (because as it is now, `IsAssignableFrom` looks like it's under `Where`, which is not right)

Reply:
Got it, done